hypothesis_sqlalchemy/hints.py

```diff
@@ -5,7 +5,7 @@
 from typing import (Tuple,
                     Union)
 
-from hypothesis.searchstrategy import SearchStrategy
+from hypothesis.strategies.internal import SearchStrategy
```

Comment:
`.internal` suggests that we should not use it, let's try `hypothesis.strategies` instead